postgres/assets/configuration/spec.yaml

```diff
@@ -177,6 +177,24 @@ files:
       value:
         type: string
         example: "datadog-agent"
+    - name: deep_database_monitoring
+      description: |
+        Set to `true` to enable the BETA features for Deep Database Monitoring.
+
+        If you would like to hear more about Deep Database Monitoring, please reach out to your customer
+        success manager or Datadog support.
+      value:
+        type: boolean
+        example: true
+        default: false
+    - name: pg_stat_statements_view
+      description: |
+        Set this value if you want to define a custom view or function to allow the datadog user to query the
+        `pg_stat_statements` table, which is useful for restricting the permissions given to the datadog agent.
+        Please note this is an experimental feature and is subject to change or deprecation without notice.
+      value:
+        type: string
+        example: show_pg_stat_statements()
```

Comment:
Missing `statement_metric_limits` option

Reply:
This was intentional; is it possible to leave the limits out of the spec? That's not really something we want to be advertising to users as something they can discover and raise on their own (potentially sending us a lot more data than we are expecting). We do want to leave it configurable so that we can work with customers to adjust it on a case-by-case basis without having to do a release. But ultimately we would want to work with them on that.

Reply:
On second thought, I'm going to remove this from config for now and just leave the hardcoded defaults. The current product requirement is "no limits except for spam."